lib/msal-angular/package.json

```diff
@@ -21,40 +21,37 @@
   "main": "./dist/bundles/azure-msal-angular.umd.js",
   "typings": "./dist/azure-msal-angular.d.ts",
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.1000.8",
-    "@angular-devkit/build-ng-packagr": "~0.1000.8",
-    "@angular/animations": "~10.0.9",
-    "@angular/cli": "~10.0.6",
-    "@angular/common": "~10.0.9",
-    "@angular/compiler": "~10.0.9",
-    "@angular/compiler-cli": "~10.0.9",
-    "@angular/core": "~10.0.9",
-    "@angular/forms": "~10.0.9",
-    "@angular/platform-browser": "~10.0.9",
-    "@angular/platform-browser-dynamic": "~10.0.9",
-    "@angular/router": "~10.0.9",
+    "@angular-devkit/build-angular": "~0.1102.11",
+    "@angular/animations": "~11.2.12",
+    "@angular/cli": "~11.2.11",
+    "@angular/common": "~11.2.12",
+    "@angular/compiler": "~11.2.12",
+    "@angular/compiler-cli": "~11.2.12",
+    "@angular/core": "~11.2.12",
+    "@angular/forms": "~11.2.12",
+    "@angular/platform-browser": "~11.2.12",
+    "@angular/platform-browser-dynamic": "~11.2.12",
+    "@angular/router": "~11.2.12",
     "@azure/msal-browser": "^2.14.1",
-    "@types/jasmine": "~3.5.0",
+    "@types/jasmine": "~3.6.0",
     "@types/jasminewd2": "~2.0.3",
     "@types/node": "^12.11.1",
-    "jasmine-core": "~3.5.0",
+    "jasmine-core": "~3.6.0",
     "jasmine-spec-reporter": "~5.0.0",
-    "karma": "~5.0.0",
+    "karma": "~6.3.2",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coverage-istanbul-reporter": "~3.0.2",
-    "karma-jasmine": "~3.3.0",
+    "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "^1.5.0",
-    "ng-packagr": "^10.0.0",
+    "ng-packagr": "^11.2.4",
     "protractor": "~7.0.0",
     "rxjs": "~6.5.5",
     "ts-node": "~8.3.0",
     "tslib": "^2.0.0",
-    "typescript": "~3.9.5",
+    "typescript": "~4.1.5",
     "zone.js": "~0.10.3"
   },
   "peerDependencies": {
-    "@angular/common": "9 - 12",
-    "@angular/core": "9 - 12",
```

Comment:
Why was this removed? (this essentially documents which versions of Angular we support)